### PR TITLE
chore: update ownership for anicholson DAGs

### PR DESCRIPTION
**Pull request by ANich**
## Description

This PR updates ownership for anicholson DAGs

## Related Tickets & Documents
...

### CODE REVIEW

File: dags/bqetl_backfill_complete.py [hunk 1]
```diff
@@ -17,7 +17,6 @@
 
 default_args = {
     "email": [
-        "anicholson@mozilla.com",
```
Comment (scholtzan) [inline dags/bqetl_backfill_complete.py:20]:
```suggestion
        "ascholtz@mozilla.com",
```

File: dags/bqetl_backfill_initiate.py [hunk 2]
```diff
@@ -17,7 +17,6 @@
 
 default_args = {
     "email": [
-        "anicholson@mozilla.com",
```
Comment (scholtzan) [inline dags/bqetl_backfill_initiate.py:20]:
```suggestion
        "ascholtz@mozilla.com",
```

File: dags/bqetl_backfill_complete.py (hunk 1 above)
Comment (ANich) [reply dags/bqetl_backfill_complete.py:20]: GH won't let me commit suggestions on a delete interestingly

Comment (scholtzan) [reply dags/bqetl_backfill_complete.py:20]: huh, interesting

**Review by sean-rose — APPROVED** [review]

**Pull request merged**